### PR TITLE
Add @LeonvanZijl

### PR DESCRIPTION
**Pull request by LeonvanZijl**

### CODE REVIEW

File: site/class-1-load/leonvanzijl/leonvanzijl/index.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+day,dinner
```
Comment (wooorm) [inline site/class-1-load/leonvanzijl/leonvanzijl/index.csv]: You’re uploading your files in `site/class-1-load/leonvanzijl/leonvanzijl/` but they should be added to `site/class-1-load/leonvanzijl/`.

Comment (wooorm) [reply site/class-1-load/leonvanzijl/leonvanzijl/index.csv]: ...and use absolute dates in your CSV!

Comment (LeonvanZijl) [reply site/class-1-load/leonvanzijl/leonvanzijl/index.csv]: uploaded the files directly in site/class-1-load/leonvanzijl/ and changed the dates in the CSV / Json

Comment (wooorm) [reply site/class-1-load/leonvanzijl/leonvanzijl/index.csv]: That looks great, but you still have the extra directory. Maybe remove this branch and try again?

**Comment by wooorm** [comment]
@LeonvanZijl Could you add `dinners.txt` too? And add yourself to the pinned post on Slack?

**Comment by wooorm** [comment]
Still missing `dinners.txt` (the file you wrote your initial dataset in)!

**Pull request merged**